athena-dynamodb/src/main/java/com/amazonaws/athena/connectors/dynamodb/resolver/DynamoDBFieldResolver.java

```diff
@@ -82,6 +82,10 @@ public Object getFieldValue(Field field, Object originalValue)
                 return DDBTypeUtils.coerceValueToExpectedType(fieldValue, field, fieldType, metadata);
         }
 
+        if (fieldValue == null) {
+            return null;
+        }
+
```

Comment:
Can you move this check earlier? To like under line 70?

Reply:
Sure! Done!